charts/kommander-operator/templates/rback_v1_clusterrole_operator_kommander-operator.yaml

```diff
@@ -113,13 +113,26 @@ rules:
       - helm.toolkit.fluxcd.io
       - kustomize.toolkit.fluxcd.io
       - apps.kommander.d2iq.io
+      - rbac.authorization.k8s.io
     resources:
       - events
       - configmaps
       - secrets
       - helmreleases
       - kustomizations
       - appdeployments
+      - clusterrolebindings
     verbs:
       - create
+  # cluster-admin permissions needed here
```

Comment:
why do we need cluster-admin?

Reply:
because now the kommander-operator needs to create a CRB that binds the default user with `cluster-admin` CR, and without this permissions, it is a permission escalation and k8s API server denies the CRB creation. right not with these changes KUTTL passed, except bootstrapper Kuttl.